mesonbuild/dependencies/framework.py

```diff
@@ -89,7 +89,7 @@ def _get_framework_path(self, path: str, name: str) -> T.Optional[Path]:
         return None
 
     def _get_framework_latest_version(self, path: Path) -> str:
-        versions = []
+        versions: list[Version] = []
```

Comment:
Meson supports Python 3.7. Is this syntax supported? Typically it is `T.List[Version]`

Reply:
Ah, no, this made it past because I run 3.10 and 3.11. Will fix now.